pkg/innerring/processors/reputation/process_put.go

```diff
@@ -48,3 +63,18 @@ func (rp *Processor) processPut(epoch uint64, id reputation.PeerID, value reputa
 			zap.String("error", err.Error()))
 	}
 }
+
+func (rp *Processor) checkManagers(e uint64, mng apireputation.PeerID, peer apireputation.PeerID) error {
+	mm, err := rp.mngBuilder.BuildManagers(e, reputation.PeerIDFromBytes(peer.ToV2().GetPublicKey()))
+	if err != nil || len(mm) == 0 {
+		return errCouldNotBuildManagers
```

Comment:
We do not recognize what the `err` is. I suggest to return `bool` from the method, and log this error.

Reply:
I mean to return only `bool` from the method. Here is the only place for the error, an we log it.

We can skip check the emptiness of managers, because the code below will eventually fail the check.

Reply:
> Here is the only place for the error, an we log it.

"manager not found" is also an error case. so one check function that returns any kind of error and logging the result of the checking after that vs logging inside after every mistake and returning bool. what should we do with that bool? one more check to do what? or just `return`?

> We can skip check the emptiness of managers, because the code below will eventually fail the check.

but what are the cons of the additional information about reasons for error? there somehow can not be the peer in the netmap at all, or the peer can have managers that are different from received one